project/overview/share.rst

```diff
@@ -0,0 +1,25 @@
+========================
+Share Projects and Tasks
+========================
+
+When you share your projects/tasks, you allow external users to be on the same page as you.
+It avoids misunderstandings, simplifies communication, and allows you to become a more coordinated
+company as you get the right information to the right people.
+
```

Comment:
I actually struggle to see how the shared screen looks like with runbot. I tried on my own db but there is immediately a redirect that makes it not work properly. Could you add an image that illustrate what the shared project looks like on the front end?  
Just a suggestion, it is not really that necessary.

Cheers :)